src/spec/integration_support/config_server_service.rb

```diff
@@ -37,10 +36,11 @@ def initialize(port_provider, base_log_path, logger, test_env_number)
     end
 
     def self.install
+      binary_file_path = ENV.fetch('CONFIG_SERVER_BINARY')
```

Comment:
It might be helpful to pull usages of `ENV` up to `Sandbox` or higher so that it is easier to reason about.